_core/smart/tutorial.md

```diff
@@ -1,211 +1,188 @@
+The following set of commands will achieve the same goals as steps 1-4.
 
-### Deploying contract
+```shell
+# Install early release of the new Blockstack CLI for Stacks 2.0
+sudo npm install -g "https://github.com/blockstack/cli-blockstack#feature/stacks-2.0-tx"
 
-Further down in the file, you find a contract deployment:
+# Create a new STX address and save keychain details to a file
+blockstack make_keychain -t > new_keychain.txt
 
-```js
-await helloWorldClient.deployContract();
-```
+# Call faucet to get STX tokens, replace <stx_address>
+curl -XPOST "https://sidecar.staging.blockstack.xyz/sidecar/v1/debug/faucet?address=<stx_address>" | json_pp
 
-### Run public functions
+# Confirm faucet transaction, replace <stx_address>
+blockstack balance <stx_address> -t
 
-Finally, you will find snippets that call the public `say-hi` function of the contract:
+# Deploy a contract file, replace <stx_private_key>
+blockstack deploy_contract ./hello-world.clar hello-world 2000 0 <stx_private_key> -t
 
-```js
-const query = helloWorldClient.createQuery({ function: { name: "say-hi", args: [] } });
-const receipt = await helloWorldClient.submitQuery(query);
-const result = Result.unwrapString(receipt);
+# Call increment method of deployed contract, replace <stx_address> and <stx_private_key>
+blockstack call_contract_func <stx_address> hello-world echo-number 2000 1 <stx_private_key> -t
 ```
 
-As you see, smart contract calls are realized through query definitions. The `createQuery` function defines the name and arguments passed to the smart contract function. With `submitQuery`, the function is executed and the response is wrapped into a `Result` object. To obtain the readable result, we use the `unwrapString` function, which should return `hello world`.
+To learn more about the Blockstack CLI commands, you can run `blockstack-cli help all`.
 
-Now, review the last test `should echo number` on your own and try to understand how arguments are passed to the `echo-number` smart contract.
+---
 
-With that, you have completed the first Clarity tutorial! In just a few minutes you ...
+With the completion of this tutorial, you ...
 
 * Created a working Clarity starter project
 * Understood basic Clarity language design principles
-* Understood how to interact with smart contracts
-* Understood how to test smart contracts
+* Deployed a smart contract to the Testnet
+* Called public methods of the deployed smart contract
+* Understood how to use the Explorer Sandbox functionality
+* Understood how to use the Blockstack CLI for advanced capabilities
 
 Congratulations!
 
 ## Where to go next
 
 {:.no_toc}
 
+* <a href="principals.html">Guide: Understanding principals</a>
 * <a href="tutorial-counter.html">Next tutorial: Writing a counter smart contract</a>
+* <a href="principals.html">Guide: Testing contracts with JavaScript and Mocha</a>
```

Comment:
the link is wrong. it's also labeled as a guide but listed in the tutorial nav section

Reply:
That's a good point; honing it down to two links here